### PR TITLE
[Queues] Add Queues bindings to Wrangler configuration page

### PR DESCRIPTION
**Pull request by kewbish**
This PR adds the Queues bindings from [here](https://developers.cloudflare.com/queues/platform/configuration/) to the Wrangler configuration page as well.

### CODE REVIEW

**Comment by kewbish** [comment]
(The inclusive language check failure wasn't part of my PR ^)

File: content/workers/wrangler/configuration.md [hunk 1]
```diff
@@ -442,6 +442,82 @@ kv_namespaces = [
 ]
 ```
 
+### Queues
+
+[Queues](/queues/) is Cloudflare's global message queueing service, providing guaranteed delivery and message batching. To interact with a Queue with Workers, you need a producer Worker to send messages to the Queue and a consumer Worker to pull batches of messages out of the Queue. A single Worker can produce to and consume from multiple Queues.
```
Comment (Maddy-Cloudflare) [inline content/workers/wrangler/configuration.md]:
```suggestion
[Queues](/queues/) is Cloudflare's global message queueing service, providing [guaranteed delivery](/queues/learning/delivery-guarantees/) and [message batching](/queues/learning/batching-retries/). To interact with a queue with Workers, you need a producer Worker to send messages to the queue and a consumer Worker to pull batches of messages out of the Queue. A single Worker can produce to and consume from multiple Queues.
```

File: content/workers/wrangler/configuration.md [hunk 2]
```diff
@@ -442,6 +442,82 @@ kv_namespaces = [
...
+
+To bind Queues to your producer Worker, assign an array of the below object to the `[[queues.producers]]` key.
+
+{{<definitions>}}
+
+- `queue` {{<type>}}string{{</type>}} {{<prop-meta>}}required{{</prop-meta>}}
+
+  - The name of the Queue, used on the Cloudflare dashboard.
```
Comment (Maddy-Cloudflare) [inline content/workers/wrangler/configuration.md]:
```suggestion
  - The name of the queue, used on the Cloudflare dashboard.
```

File: content/workers/wrangler/configuration.md [hunk 3]
```diff
@@ -442,6 +442,82 @@ kv_namespaces = [
...
+
+- `binding` {{<type>}}string{{</type>}} {{<prop-meta>}}required{{</prop-meta>}}
+
+  - The binding name used to refer to the Queue in your Worker. The binding must be [a valid JavaScript variable name](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#variables). For example, `binding = "MY_QUEUE"` or `binding = "productionQueue"` would both be valid names for the binding.
```
Comment (Maddy-Cloudflare) [inline content/workers/wrangler/configuration.md]:
```suggestion
  - The binding name used to refer to the queue in your Worker. The binding must be [a valid JavaScript variable name](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#variables). For example, `binding = "MY_QUEUE"` or `binding = "productionQueue"` would both be valid names for the binding.
```

File: content/workers/wrangler/configuration.md [hunk 4]
```diff
@@ -442,6 +442,82 @@ kv_namespaces = [
...
+  binding = "MY_QUEUE"
+```
+
+To bind Queues to your consumer Worker, assign an array of the below object to the `[[queues.consumers]]` key.
+
+{{<definitions>}}
+
+- `queue` {{<type>}}string{{</type>}} {{<prop-meta>}}required{{</prop-meta>}}
+
+  - The name of the Queue, used on the Cloudflare dashboard.
```
Comment (Maddy-Cloudflare) [inline content/workers/wrangler/configuration.md]:
```suggestion
  - The name of the queue, used on the Cloudflare dashboard.
```

File: content/workers/wrangler/configuration.md [hunk 5]
```diff
@@ -442,6 +442,82 @@ kv_namespaces = [
...
+
+  - The maximum number of seconds to wait for messages to fill a batch before the batch is sent to the consumer Worker.
+
+- `max_retries` {{<type>}}number{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
+
+  - The maximum number of retries for a message, if it fails or [`retryAll()`](/queues/platform/javascript-apis/#messagebatch) is invoked.
+
+- `dead_letter_queue` {{<type>}}string{{</type>}} {{<prop-meta>}}optional{{</prop-meta>}}
+
+  - The name of another Queue to send a message if it fails processing at least `max_retries` times.
```
Comment (Maddy-Cloudflare) [inline content/workers/wrangler/configuration.md]:
```suggestion
  - The name of another queue to send a message if it fails processing at least `max_retries` times.
```

File: content/workers/wrangler/configuration.md [hunk 6]
```diff
@@ -442,6 +442,82 @@ kv_namespaces = [
...
+  - If a dead_letter_queue is not defined, messages that repeatedly fail processing will be discarded.
+  - If there is no Queue with the specified name, it will be created automatically.
```
Comment (Maddy-Cloudflare) [inline content/workers/wrangler/configuration.md]:
```suggestion
  - If there is no queue with the specified name, it will be created automatically.
```

**Review by Maddy-Cloudflare — APPROVED** [review]
@kewbish  wait for @deadlypants1973 to review this before merging.

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**